### PR TITLE
init update for ruby 3.x

### PR DESCRIPTION
**Pull request by wheatley**
## Category
Maintainability 

## Feature/Issue Description

Initial PR for updating to ruby 3.x 

## Test Cases

Ran successfully unit and integration tests

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -252,7 +260,7 @@ GEM
       nokogiri (~> 1.8)
 
 PLATFORMS
-  ruby
+  x86_64-linux
```
Comment (bcoles) [inline Gemfile.lock:263]: ?

File: Gemfile.lock [hunk 2]
```diff
@@ -92,29 +95,33 @@ GEM
...
+      timeout
+    net-smtp (0.3.1)
+      digest
+      net-protocol
+      timeout
     netrc (0.11.0)
-    nio4r (2.5.8)
-    nokogiri (1.12.5)
-      mini_portile2 (~> 2.6.1)
+    nokogiri (1.13.3-x86_64-linux)
```
Comment (bcoles) [inline Gemfile.lock:117]: ?

Comment (wheatley) [reply Gemfile.lock:117]: this is what installed by default when creating the .lock

File: Gemfile.lock (hunk 1 above)
Comment (wheatley) [reply Gemfile.lock:263]: again, this is what was set when creating the .lock file

**Pull request merged**